Algorithms/Greedy/Kruskal.cpp

```diff
@@ -0,0 +1,87 @@
+//Name:Dhiraj Sharma
+//Email-id:dhirajcoolssharma@gmail.com
+// C++ program for Kruskal's algorithm to find Minimum Spanning Tree
+#include<bits/stdc++.h>
+using namespace std;
+//Need to create an undirected weighted and ordered graph.
+//Hence write the starting and ending vertices to make an edge,
+//this edge has to a weighted edge.
+class Edge
+{   public:
+        int start_vertex;
+        int end_vertex;
+        int weight;
+};
+//A comparision function for STL based sorting in increasing order of weight
+bool mycomp(Edge e1, Edge e2)
+{
+    return e1.weight<e2.weight ;
+}
+//This function provides the start and end vertex parent of an edge
+int find(int v, int* parent)
+{
+    if(parent[v]==v)
+        return v;
+    return find(parent[v],parent);
```

Comment:
Add path compression to reduce time complexity.

Reply:
Path compression is a modification in Union Find itself. Complete that within this issue itself. It's literally changing 1 operator in your code.

Reply:
Okay,as you wish.Will update it ?